pkg/overlay/cache_test.go

```diff
@@ -66,7 +66,7 @@ func newTestKademlia(t *testing.T, ip, port string, d dht.DHT, b pb.Node) *kadem
 	assert.NoError(t, err)
 	identity, err := ca.NewIdentity()
 	assert.NoError(t, err)
-	kad, err := kademlia.NewKademlia(&id, n, fmt.Sprintf("%s:%s", ip, port), identity, "db", kc)
+	kad, err := kademlia.NewKademlia(&id, n, net.JoinHostPort(ip, port), identity, "db", kc)
```

Comment:
Is there a reason it's taking a ip/port at all? I mean for tests it always should be `127.0.0.1:0`.

Reply:
There appears to be some holistic issues here, so we've broken that out into another issue:
https://storjlabs.atlassian.net/browse/V3-533